grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -75,6 +81,24 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
       alertReceiveChannelStore.getSearchResult() && alertReceiveChannelStore.getSearchResult()[0];
     if (directPagingAlertReceiveChannel) {
       setSelectedTeamDirectPaging(directPagingAlertReceiveChannel);
+      await alertReceiveChannelStore.updateChannelFilters(directPagingAlertReceiveChannel.id);
+      await store.slackChannelStore.updateItems();
+
+      // Set unique available chatops channels
+      const filterIds = alertReceiveChannelStore.channelFilterIds[directPagingAlertReceiveChannel.id];
+      let availableChannels = [];
+      let channelKeys = new Set();
+      filterIds.map((channelFilterId) => {
+        IntegrationHelper.getChatOpsChannels(alertReceiveChannelStore.channelFilters[channelFilterId], store)
+          .filter((channel) => channel)
```

Comment:
weird that we need to filter, why might they be nullable? maybe we need to filter them on backend side....